From: Markus Koschany <apo@gambaru.de>
Date: Sun, 30 Jun 2013 19:13:54 +0200
Subject: trim libraries in sane-backends.pc.in

Update the original sane-config_and_pkg-config_fixes.patch from Julien BLACHE.
Patching RPATH related flags is no longer necessary. Still trim the libraries
in sane-backends.pc.in to the bare minimum for what is needed for libsane.
---
 tools/sane-backends.pc.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/tools/sane-backends.pc.in
+++ b/tools/sane-backends.pc.in
@@ -2,8 +2,8 @@
 exec_prefix=@exec_prefix@
 libdir=@libdir@
 includedir=@includedir@
-ldflags=@LDFLAGS@ @GPHOTO2_LDFLAGS@
-libs=@LIBS@ @DL_LIBS@ @LIBV4L_LIBS@ @MATH_LIB@ @TIFF_LIBS@ @JPEG_LIBS@ @GPHOTO2_LIBS@ @SOCKET_LIBS@ @AVAHI_LIBS@ @USB_LIBS@ @SCSI_LIBS@ @RESMGR_LIBS@
+ldflags=@LDFLAGS@
+libs=@LIBS@
 
 Name: SANE Backends
 Description: Backends for SANE, the universal scanner interface
